persistence/pagination/Page_test.go

```diff
@@ -0,0 +1,44 @@
+package pagination_test
+
+import (
+	"testing"
+
+	"github.com/ditrit/badaas/persistence/pagination"
+	"github.com/stretchr/testify/assert"
+)
+
+type Whatever struct {
+	DumbData int
+}
+
+func (Whatever) TableName() string {
+	return "whatevers"
+}
+
+func TestNewPage(t *testing.T) {
```

Comment:
add more tests, one test for each next point:
- check lastPage false
- check lastPage true
- check IsFirstPage false
- check IsFirstPage true
- check IsFirstPage false
- check HasPreviousPage true
- check HasPreviousPage false
- check HasNextPage true
- check HasNextPage false
- check HasContent true
- check HasContent false